tests/foreman/ui/test_syncplan.py

```diff
@@ -643,8 +655,8 @@ def test_positive_synchronize_custom_products_future_sync_date(self):
             #  Verify products have not been synced yet
             for repo in repos:
                 self.validate_repo_content(
-                    repo.product.read().name,
-                    repo.name,
+                    repo.product,
```

Comment:
Hey guys, sorry for delay, I'm doing the read operation now in `validate_repo_content()` (lines 80, 81), checked with:
```
py.test tests/foreman/ui/test_syncplan.py -k test_positive_synchronize_custom_product_future_sync_date
========================================== test session starts ==========================================
platform linux2 -- Python 2.7.13, pytest-3.1.3, py-1.4.34, pluggy-0.4.0
rootdir: /home/pondrejk/Documents/robottelo, inifile:
plugins: xdist-1.19.1, services-1.2.1, mock-1.6.2, forked-0.2, cov-2.5.1
collected 21 items 
2017-08-18 13:22:17 - conftest - DEBUG - Deselect of WONTFIX BZs is disabled in settings


tests/foreman/ui/test_syncplan.py .

========================================== 20 tests deselected ==========================================
=============================== 1 passed, 20 deselected in 678.29 seconds ===============================
```